<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessBox</class>
 <widget class="QDialog" name="ProcessBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProcessBox</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>190</width>
     <height>40</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="param1SpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>2.000000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>2.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="param1Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Min plane area:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="param2SpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>300.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5000.000000000000000</double>
   </property>
   <property name="value">
    <double>600.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="param2Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Distance threshold:</string>
   </property>
  </widget>
  <widget class="QLabel" name="param3Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Max interations</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="param3SpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>300.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5000.000000000000000</double>
   </property>
   <property name="value">
    <double>400.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="param1Unit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>m2</string>
   </property>
  </widget>
  <widget class="QLabel" name="param2Unit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>millimeter</string>
   </property>
  </widget>
  <widget class="QLabel" name="param3Unit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>millimeter</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="param4SpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>300.000000000000000</double>
   </property>
   <property name="maximum">
    <double>5000.000000000000000</double>
   </property>
   <property name="value">
    <double>400.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="param4Unit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>millimeter</string>
   </property>
  </widget>
  <widget class="QLabel" name="param4Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cluster threshold</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
